services/net/contentmigration/Config/ClipMigrationOptions.cs

```diff
@@ -0,0 +1,37 @@
+using TNO.Services.Config;
+
+namespace TNO.Services.ContentMigration.Config;
+
+/// <summary>
+/// ContentMigrationOptions class, configuration options for Import
+/// </summary>
+public class ClipMigrationOptions : IngestServiceOptions
```

Comment:
Not sure I see value in having multiple identical options.  Just make a single configuration option for the service.

Reply:
I would love to do this but I couldnt figure out the correct way to get DependencyInjection to work with the same "Options" being used by different migrators.  If you take a look at the `ContentMigratorFactoryExtensions.cs` you will see what I mean.

Reply:
You can do this a few different ways.  However, the main way is through configuration through the appsettings.jons file, or ENV variables.  Are you running each type of the Content Migration Service in a single instance?  Or will we run four different instances?  If you are doing it all in one instances then you can separate the configuration and dependency injection by passing a different path to the configuration.

```
{
  "Service": {
    "Snippet": {
      "key": "value"
    },
    "Image": {
      "key": "value"
    }
  }
}
```

Reply:
If you're doing this with four different instances then you don't need to have a different configuration path for each time.